MediaPlayer/MediaControls.js

```diff
@@ -163,6 +163,14 @@ const MediaControlsBase = kind({
 		 */
 		onClose: PropTypes.func,
 
+		/**
+		 * Called when the user flicks on the action guide.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		onFlickFromActionGuide: PropTypes.func,
```

Comment:
I think it should be private. (with handleKeyDownFromMediaButtons?)  But I'm not sure
@MikyungKim Plese decide it

Reply:
Let's make it private.